nautobot_ssot/integrations/ipfabric/diffsync/adapter_ipfabric.py

```diff
@@ -0,0 +1,161 @@
+# pylint: disable=duplicate-code
+"""DiffSync adapter class for Ip Fabric."""
+
+import logging
+
+from diffsync import ObjectAlreadyExists
+from django.conf import settings
+from nautobot.dcim.models import Device
+from nautobot.ipam.models import VLAN
+from netutils.mac import mac_to_format
+
+from nautobot_ssot.integrations.ipfabric.diffsync import DiffSyncModelAdapters
+
+logger = logging.getLogger("nautobot.jobs")
+
+CONFIG = settings.PLUGINS_CONFIG.get("nautobot_ssot", {})
+DEFAULT_INTERFACE_TYPE = CONFIG.get("default_interface_type", "1000base-t")
+DEFAULT_INTERFACE_MTU = CONFIG.get("default_interface_mtu", 1500)
+DEFAULT_INTERFACE_MAC = CONFIG.get("default_interface_mac", "00:00:00:00:00:01")
+DEFAULT_DEVICE_ROLE = CONFIG.get("default_device_role", "Network Device")
+DEFAULT_DEVICE_STATUS = CONFIG.get("default_device_status", "Active")
```

Comment:
These config keys are not present in nautobot_config.py neither in PluginConfig default keys.